adoc/Release-Notes.adoc

```diff
@@ -33,6 +33,174 @@ endif::[]
 [id='sec.major-change']
 == Major Changes
 
+[id='sec.1_5']
+=== Release 1.5, September 2019
+
+[id='sec.1_5.new']
+==== What Is New?
+
+* {scf} has been updated to version 2.18.0:
+** PXC (Percona XtraDB Cluster) replaces `cf-mysql` for database management -- please read the Known Issues section for this version on deployment and upgrade changes
+** Ability to set `config.HA_strict=false` in combination with `config.HA=true` to allow lowering the sizing count for a role below what is required for HA
+** UAA can now be embedded in the `cf` namespace -- please read the Known Issues section for this version on deployment limitations
+* The Stratos UI has been updated to version 2.5.1:
+** Tech preview for helm feature
+** Add custom welcome message on endpoints page
+** Add support for connecting {caasp} V4 endpoints
+** Refinements to the Autoscaler UI
+** For a full list of features and fixes see https://github.com/SUSE/stratos/releases/tag/2.5.1.
+
+For information about deploying and administering {product}, see the product manuals at
+{doc-url}.
+
+[id='sec.1_5.feature']
+==== Features and Fixes
+* `cf-deployment` has been updated to version 9.5.0.
+* Eirini updated to 0.0.14
+* Removed the `cluster-admin` role binding for the eirini service account
+* Switched over to `PXC` from `cf-mysql` for database management
+* Includes these {cf} component versions:
+** app-autoscaler: 1.2.1
+** bits-service: 2.28.0
+** bpm: 1.1.0
+** capi: 1.83.0
+** cf-deployment: 9.5
+** cf-mysql: 36.15.0
+** cf-routing: 0.188.0
+** cf-sle12: 1.81.26
+** cf-sle15: 10.28
+** cf-smoke-tests: 40.0.112
+** cf-syslog-drain: 10.2
+** cf-usb: 1.0.1
+** cflinuxfs3: 0.118.0
+** credhub: 2.4.0
+** diego: 2.34.0
+** eirini: 0.0.14
+** garden-runc: 1.19.3
+** groot-btrfs: 1.0.5
+** log-cache: 2.2.2
+** loggregator: 105.5
+** loggregator-agent: 3.9
+** mapfs: 1.1.0
+** nats: 27
+** nfs-volume: 1.5.2
+** postgres-release: 26
+** pxc: 0.18.0
+** scf-helper: 1.0.3
+** statsd-injector: 1.10.0
+** uaa: 72.0
+* Buildpacks:
+** binary-buildpack: 1.0.33
+** dotnet-core-buildpack: 2.2.13
+** go-buildpack: 1.8.42
+** java-buildpack: 4.20.0 
+** nginx-buildpack: 1.0.15
+** nodejs-buildpack: 1.6.53
+** php-buildpack: 4.3.80
+** python-buildpack: 1.6.36
+** ruby-buildpack: 1.7.42
+** staticfile-buildpack: 1.4.43
+
+[id='sec.1_5.issue']
+==== Known Issues
+
+IMPORTANT: In order to deploy this version in HA, or upgrade from {cap} 1.4.1 from HA, you will need to scale down the `mysql` role to 1 in order to migrate from cf-mysql to PXC. This is based on upstream instructions but based on what we've seen with other components that rely on the database, such as Credhub, scaling all database roles into single availability helps with a stable migration and deployment.
```

Comment:
Can we reword this section as:
In order to deploy {cap} 1.5 or upgrade from {cap} 1.4.1 in an HA configuration you will first need to scale the `mysql` role down to 1 instance to be able to migrate from `cf-mysql` to PXC.

Reply:
We can.